language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/List.hs

```diff
@@ -169,52 +160,6 @@ getBuiltinFoldList = do
           , Var () xs'
           ]
 
--- | @enumFromCount@
---
--- > enumFromCount :: Nat -> Nat -> [Nat]
--- > enumFromCount n zero    = []
--- > enumFromCount n (suc k) = n : enumFromCount (suc n) k
---
--- as a PLC term
---
--- > \(n k : nat) ->
--- >     foldrNat {nat -> list nat}
--- >         (\(rec : nat -> list nat) (n' : nat) -> cons {nat} n' (rec (succ n')))
--- >         (\(n' : nat) -> nil {nat})
--- >         k
--- >         n
---
--- @n'@ appears twice in the term, @nat@ appears several times in types,
--- so this is not really a definition with unique names.
-getBuiltinEnumCountNat :: Quote (Term TyName Name ())
```

Comment:
why did you delete this?

Reply:
> I also removed `getBuiltinEnumCountNat`. It doesn't really belong to the stdlib. I think, I'll return it later in the form of an "interesting generator", but for now I'm just removing this noise.

Reply:
:sweat_smile: I can read, I promise

Reply:
I thought that maybe you just missed that sentence (happens to me). That's not an stdlib function, it doesn't do anything useful. I planned to implement `enumFromTo` in terms of it, but then realized I better stop avoiding dealing with `integer`s. This function is a noise, so I removed it.